lib/eChronicler/e_chronicler/journal_entry.ex

```diff
@@ -0,0 +1,58 @@
+defmodule EChronicler.EChronicler.JournalEntry do
```

Comment:
Okay, first of all, Elixir style guide recommends using `snake_case` file names for `TitleCase` modules.

So this would mean: `e_chronicler/e_chronicler/journal_entry.ex` for a module named `EChronicler.EChronicler.JournalEntry`.

That being said...I don't really like the duplicate naming of `EChronicler.EChronicler`. I would go for something like `EChronicler.Models.JournalEntry`. This would mirror how the controllers and views are set up too.

Reply:
Reminder to read through the Elixir style guide here: https://github.com/christopheradams/elixir_style_guide